src/middleware/entryIssueDetails.middleware.js

```diff
@@ -46,7 +46,7 @@ const fetchIssueCount = fetchOne({
     select count(*) as count
     from issue i
     LEFT JOIN issue_type it ON i.issue_type = it.issue_type
-    WHERE resource = '${req.resources[0].resource}'
+    WHERE resource = '${req.resources[0]?.resource || ''}'
```

Comment:
Actually this will return all resources with empty - we probably want to return 0 values